drivers/ukconsole/ns16550/Config.uk

```diff
@@ -1,150 +1,149 @@
-menuconfig LIBUKCONSOLE_NS16550
+menuconfig LIBNS16550
 	bool "NS16550 / 8250"
 	depends on (ARCH_ARM_64 || ARCH_X86_64)
+	depends on (!PLAT_XEN && ARCH_X86_64)
```

Comment:
So since this is replaced in a following commit with `HAVE_IBMPC_NS16550` this intermediate step should probably doesn't need to be in the history.

Reply:
An artifact of building a history that is commit-wise "bisectable". A preference of style IMO.

Reply:
Hmmm okay, let it be then.